aiohttp/http_parser.py

```diff
@@ -154,7 +154,7 @@ def parse_headers(
             if len(bname) > self.max_field_size:
                 raise LineTooLong(
                     "request header name {}".format(
-                        bname.decode("utf8", "xmlcharrefreplace")
+                        bname.decode("utf8", "backslashreplace")
```

Comment:
To clarify what the change does, this results in any invalid utf-8 bytes being transformed into literal characters:
```
>>> b"\xd9".decode("utf-8", "backslashreplace")
'\\xd9'
```
So, the invalid continuation byte becomes a literal `\xd9`, instead of 1 byte.

No idea if this is the best option, but seems better than the error produced by the current code.